packages/components/table/src/table.js

```diff
@@ -0,0 +1,148 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './table.css';
+import '@tradeshift/elements.status';
+import '@tradeshift/elements.document-card';
+import { VISIBILITY } from './utils/constants';
+
+customElementDefineHelper(
+	'ts-table',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				rtl: { type: Boolean, reflect: true },
```

Comment:
In new components we are use the `dir` property instead of `rtl`. Also, we have a fallback to a body `dir` property if this property not provided to the component. You can use the `header` component as example.

Reply:
ok